docs/Static_Data_Decompilation.md

```diff
@@ -0,0 +1,21 @@
+Static data is any variable that the compiler decides to permanently store in the ROM (usually ones marked with the `static` keyword in C). They get stored in the memory, and can be accessed anytime inside their scope.
+
+In a DS ROM binary these variables are stored in three sections:
+- .bss: static variables with no values when initialized (they take the value of 0 in startup)
+Example: `static int a;`
+- .data: static variables with a value when initialized. their value can change in runtime 
+Example: `static int a = 23;`
+- .rodata: static variables using the keyword `const` when initialized, they have a constant value that can't be changed.
+Example: `static const int pi = 3.141592`
+
+In the Pokeheartgold Repo these sections have been split alongside the Assembly subroutines, with each .asm file only containing static data used by it's functions.
+
+### General workflow of decomping static data
+
+1. See how the data is referenced inside functions. This will give you an idea of what the data type might be (e.g. a struct, or an u8 array)
```

Comment:
```suggestion
1. See how the data is referenced inside functions. This will give you an idea of what the data type might be (e.g. a struct, or an u8 array).
```